Makefile

```diff
@@ -115,7 +115,7 @@ test-docker: build
 # KPT_FN_RUNTIME can be set to select the desired function runtime.
 # If unspecified, the default function runtime will be used.
 test-fn-render: build
-	PATH=$(GOBIN):$(PATH) go test -v --tags=docker --run=TestFnRender/testdata/fn-render/$(T) ./e2e/
+	PATH=$(GOBIN):$(PATH) go test -v --tags=docker --run=TestFnSink/testdata/fn-sink/$(T) ./e2e/
```

Comment:
Why make this change? It doesn't look correct. It seems like we should have a separate make target `make test-fn-sink` to run the TestFnSink, and leave the `test-fn-render` target as is?

Reply:
Oh, good catch. I did not intend to commit this, it was just something I used for debugging. I will fix it.

Reply:
Fixed now.